torch/_dynamo/symbolic_convert.py

```diff
@@ -172,10 +172,15 @@ def wrapper(self: "InstructionTranslatorBase", inst: Instruction):
             reason = None
             try:
                 return inner_fn(self, inst)
-            except Unsupported as exc:
+            except Unsupported as excp:
+                if self.has_backedge():
+                    msg = "Skipping frame because there is a graph break in a for/while loop"
+                    log.debug(msg)
+                    raise exc.SkipFrame(msg)
```

Comment:
It seems we only skip frame when ```break_graph_if_unsupported```, should we do the same thing for ```generic_jump```? It still suffers from the guard on TUPLE_ITERATOR_LEN.
```
def inner(x):
    return torch.sin(x)

def fn(x):
    y = x
    for _ in range(100):
        inner(x)
        if y.sum() > 100:
            y = x * 2
        else:
            y = x + 2
    return y
```